content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md

```diff
@@ -0,0 +1,35 @@
+---
+Title:
```

Comment:
Please review another entry or template and update the metadata to match. There are several edits that are required:

- all of the content must be within single quotes
- the title and description should not begin on a separate line
- catalogcontent should include two items